src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -41,7 +41,8 @@
             [quo2.components.avatars.user-avatar :as user-avatar]
             [quo2.components.markdown.text :as text]
             [status-im.utils.utils :as utils]
-            [status-im2.contexts.chat.home.chat-list-item.view :as home.chat-list-item])
+            [status-im2.contexts.chat.home.chat-list-item.view :as home.chat-list-item]
+            [quo2.core :as quo2])
```

Comment:
`:as quo`

Reply:
or maybe we just `:as quo2` for now and it will be easier to a replace all once we finally migrated over.

@flexsurfer wdyt? 
I see you started using `:as quo` for quo2 stuff in the code base. This might get confusing while we're supporting both :D

Reply:
yes, when we move stuff to `status-im2` quo alias should be used, in old namespaces there are still might be mixed usage of new and old components, so that's fine to use quo2 there